### ISSUE TITLE
TICC_solver

### ISSUE THREAD

**Issue by bdega** [getcontacts/getcontacts#34]
Hi all,

I've just installed getcontacts and I'm trying a few tests.
Running 'get_contact_ticc.py' it turns out this error:

  File "/home/giuseppe/miniconda3/lib/python3.6/site-packages/ticc.py", line 1, in <module>
    from TICC_solver import solve
ImportError: cannot import name 'solve'

and actually in TICC_solver.py there's nothing but a simple variable called 'solve'.

Am I wrong in some way?
Thanks in advance,
Regards

G

**Comment by RasmusFonseca** [comment]
You're not wrong. This is a temporary issue TICC and we're working on it. The temporary fix is to roll back ticc to version 0.1.4:
```bash
pip uninstall ticc
pip install ticc==0.1.4
```
Sorry for this issue. I'll leave this open until we've fixed it.

**Comment by bdega** [comment]
Thanks a lot!

G
